packages/theme-addresses/helpers.js

```diff
@@ -0,0 +1,16 @@
+export function mergeObjects() {
+  var to = Object({});
+
+  for (var index = 0; index < arguments.length; index++) {
+    var nextSource = arguments[index];
+
+    if (nextSource) {
+      for (var nextKey in nextSource) {
+        if (Object.prototype.hasOwnProperty.call(nextSource, nextKey)) {
+          to[nextKey] = nextSource[nextKey];
+        }
+      }
+    }
+  }
+  return to;
```

Comment:
I think you can use `Object.assign({}, obj1, obj2)` here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Unless, we need deep copy?

Reply:
I did that for browser compatibility, yeah. :/